### PR TITLE
Update Android SDK 2.7.7

### PR DESCRIPTION
**Pull request by KostyaChuprynaHL**

### CODE REVIEW

**Review by jinglebiscuitsHL — APPROVED** [review]

**Comment by mdillavou** [comment]
@jinglebiscuitsHL Shouldn't this be using sdk 3.0.2?

File: SampleAndroid/app/src/main/java/com/vipaar/lime/samplehl/Utils.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+          BuildConfig.GALDR_API_KEY,
+          "Jason",
+          "https://www.securenvoy.com/sites/default/files/legacy-uploads/2013/10/pizza_hut_logo.jpg",
+          autoEnableCamera,
+          autoEnableMic,
+          false,
+          enableMiniview);
+    }
+
+    public static HLTheme createTheme() {
```
Comment (mdillavou) [inline SampleAndroid/app/src/main/java/com/vipaar/lime/samplehl/Utils.java:42]: Is this still valid? I thought we didn't have themes in the new UI?

File: SampleAndroid/app/src/main/java/com/vipaar/lime/samplehl/Utils.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+    public static HLCall renewSessionData(HLCall call, boolean autoEnableCamera, boolean autoEnableMic, boolean enableMiniview) {
+        return new HLCall(
+          call.getSessionId(),
+          call.getSessionToken(),
+          call.getUserToken(),
+          call.getGssUrl(),
+          BuildConfig.GALDR_API_KEY,
+          "Jason",
+          "https://www.securenvoy.com/sites/default/files/legacy-uploads/2013/10/pizza_hut_logo.jpg",
```
Comment (mdillavou) [inline SampleAndroid/app/src/main/java/com/vipaar/lime/samplehl/Utils.java:35]: This isn't valid.

File: SampleAndroid/app/build.gradle [hunk 3]
```diff
@@ -50,7 +50,7 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.vipaar.lime:hlsdk-light:2.0.1'
+    implementation 'com.vipaar.lime:hlsdk-light:2.7.7'
```
Comment (mdillavou) [inline SampleAndroid/app/build.gradle:53]: Update to 3.0.2

**Review by mdillavou — CHANGES_REQUESTED** [review]

**Comment by jinglebiscuitsHL** [comment]
please see PR #67. It contains the fixes for your comments.

File: SampleAndroid/app/src/main/java/com/vipaar/lime/samplehl/Utils.java (hunk 1 above)
Comment (jinglebiscuitsHL) [reply SampleAndroid/app/src/main/java/com/vipaar/lime/samplehl/Utils.java:42]: I never felt like we for sure for sure dropped support.

**Pull request closed without merging**